app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -192,18 +211,22 @@ const CourseCreator = createReactClass({
     if (this.props.loadingUserCourses) {
       return <div />;
     }
-    // There are three fundamental states: NewOrClone, CourseForm, and CloneChooser
+    // There are four fundamental states: NewOrClone, CourseForm, wizardForm and CloneChooser
     let showCourseForm;
     let showCloneChooser;
     let showNewOrClone;
+    let showWizardForm;
     // If user has no courses, just open the CourseForm immediately because there are no cloneable courses.
     if (this.props.cloneableCourses.length === 0) {
       showCourseForm = true;
-    // If the creator was launched from a campaign, do not offer the cloning option.
+      // If the creator was launched from a campaign, do not offer the cloning option.
     } else if (this.campaignParam()) {
       showCourseForm = true;
+    } else if (this.state.showWizardForm) {
```

Comment:
This isn't working in the open_course_creation_spec because `showCourseForm` is set to `true` in the first `if` that checks for cloneable courses (and the test user has no other courses).

Pulling out each state into its own component will help, but a quick fix will be put another conditional checking `state.showWizardForm` inside that first `if`.